plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer-v3.ts

```diff
@@ -1,525 +0,0 @@
-import { KAFKA_SESSION_RECORDING_SNAPSHOT_ITEM_EVENTS } from '../../../config/kafka-topics'
-import { BatchConsumer, startBatchConsumer } from '../../../kafka/batch-consumer'
-import { createRdConnectionConfigFromEnvVars, createRdProducerConfigFromEnvVars } from '../../../kafka/config'
-import { createKafkaProducer } from '../../../kafka/producer'
-import { PluginsServerConfig, TeamId } from '../../../types'
-import { BackgroundRefresher } from '../../../utils/background-refresher'
-import { KafkaProducerWrapper } from '../../../utils/db/kafka-producer-wrapper'
-import { PostgresRouter } from '../../../utils/db/postgres'
-import { status } from '../../../utils/status'
-import { fetchTeamTokensWithRecordings } from '../../../worker/ingestion/team-manager'
-import { expressApp } from '../../services/http-server'
-import { ObjectStorage } from '../../services/object_storage'
-import { runInstrumentedFunction } from '../../utils'
-import { addSentryBreadcrumbsEventListeners } from '../kafka-metrics'
-import { ConsoleLogsIngester } from './services/console-logs-ingester'
-import { ReplayEventsIngester } from './services/replay-events-ingester'
-import { BUCKETS_KB_WRITTEN, BUFFER_FILE_NAME, SessionManagerV3 } from './services/session-manager-v3'
-import { IncomingRecordingMessage } from './types'
-import { allSettledWithConcurrency, parseKafkaMessage, reduceRecordingMessages } from './utils'
-
-// Must require as `tsc` strips unused `import` statements and just requiring this seems to init some globals
-require('@sentry/tracing')
-
-// WARNING: Do not change this - it will essentially reset the consumer
-const KAFKA_CONSUMER_GROUP_ID = 'session-replay-ingester'
-const KAFKA_CONSUMER_SESSION_TIMEOUT_MS = 60000
-
-// NOTE: To remove once released
-const metricPrefix = 'v3_'
-
-const gaugeSessionsHandled = new Gauge({
-    name: metricPrefix + 'recording_blob_ingestion_session_manager_count',
-    help: 'A gauge of the number of sessions being handled by this blob ingestion consumer',
-})
-
-const histogramKafkaBatchSize = new Histogram({
-    name: metricPrefix + 'recording_blob_ingestion_kafka_batch_size',
-    help: 'The size of the batches we are receiving from Kafka',
-    buckets: [0, 50, 100, 250, 500, 750, 1000, 1500, 2000, 3000, Infinity],
-})
-
-const histogramKafkaBatchSizeKb = new Histogram({
-    name: metricPrefix + 'recording_blob_ingestion_kafka_batch_size_kb',
-    help: 'The size in kb of the batches we are receiving from Kafka',
-    buckets: BUCKETS_KB_WRITTEN,
-})
-
-const counterKafkaMessageReceived = new Counter({
-    name: metricPrefix + 'recording_blob_ingestion_kafka_message_received',
-    help: 'The number of messages we have received from Kafka',
-    labelNames: ['partition'],
-})
-
-export interface TeamIDWithConfig {
-    teamId: TeamId | null
-    consoleLogIngestionEnabled: boolean
-}
-
-/**
- * @deprecated Delete reduceRecordingMessages and associated tests when deleting this.
```

Comment:
Thanks! As specified here, could you please remove `reduceRecordingMessages` and its tests on this PR too?